R/get_linelist.R

```diff
@@ -0,0 +1,109 @@
+#' Get patient linelist data
+#' 
+#' @description 
+#'  `r lifecycle::badge("deprecated")`
+#'  
+#'  Provides a public international patient linelist from January 2020 to June 
+#'  2020. 
+#'  
+#'  This version of the linelist has stopped updating. The new version of the
+#'  linelist is free but requires a login.
+#'  
+#'  See: https://global.health/
+#'  
+#' @param clean Logical, defaults to `TRUE`. 
+#'  Should the data returned be cleaned for use.
+#' @param report_delay_only Logical, defaults to `FALSE`. 
+#'  Should only certain variables (id, country, onset date, days' delay), 
+#'  and observations (patients with a report delay) be returned
+#' @return A linelist of reported cases of Covid-19
+#' @source https://github.com/beoutbreakprepared/nCoV2019
+#' @author Sam Abbott <sam.abbott@lshtm.ac.uk>
+#' 
+#' @keywords internal
+#' 
+#' @importFrom lifecycle deprecate_warn
+#' @importFrom dplyr if_else select mutate filter
+#' @importFrom lubridate dmy
+#' @importFrom tibble as_tibble
+#' @importFrom utils download.file untar
+#' @examples 
+#' \dontrun{
+#' # Get the complete linelist
+#' get_linelist()
+#'
+#' # Return the report delay only
+#' get_linelist(report_delay_only = TRUE)
```

Comment:
Can we keep the archived so it continues to return something whilst we resolve? That data is not useless just out of date. Message indicating when the data is from also needs to come back.